include/wx/uilocale.h

```diff
@@ -40,6 +41,16 @@ class WXDLLIMPEXP_BASE wxLocaleIdent
     // and region separated by dashes.
     static wxLocaleIdent FromTag(const wxString& tag);
 
+    // Add likely subtags to a given locale identifier.
+    static wxLocaleIdent AddLikelySubtags(const wxLocaleIdent& localeIdent);
```

Comment:
Does this function really need to be public? It isn't documented, so I'm not exactly sure what is it for, but it looks more like an internal helper, will it really be useful to the application code?